pkg/proto/configuration/tls/tls.proto

```diff
@@ -36,14 +38,26 @@ message ClientConfiguration {
   // addresses. This field can be used to specify the expected DNS name
   // of the server certificate.
   string server_name = 5;
+
+  // x509 Key-Pair to use for client TLS. No client
+  // certificate/private key is used when left unset.
+  oneof client_key_pair {
```

Comment:
Could we move this into its own message?
```
message X509KeyPair {
   message Inline {
   }

   message Files {
   }

   oneof { ... }
}
```
That way client and server certificates will have the same schema. I guess that would simplify the code a bit as well?

Reply:
Done!